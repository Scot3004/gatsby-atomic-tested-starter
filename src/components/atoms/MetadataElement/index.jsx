import styled from 'styled-components';

export const MetadataElement = styled.div`
  color: #555;
  display: inline-block;
  margin: 0;
`;

export default MetadataElement;
