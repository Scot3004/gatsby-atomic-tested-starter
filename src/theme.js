const theme = {
  backgroundColor: '#f5f5f5',
  textColor: '#000',
  mutedColor: '#555',
  primaryColor: '#663399',
  visitedColor: '#402060'
};

export default theme;
