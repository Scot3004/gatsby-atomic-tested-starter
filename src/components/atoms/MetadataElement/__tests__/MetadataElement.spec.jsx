import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import theme from '../../../../theme';
import MetadataElement from '..';

describe('When I see the metadata of a post', () => {
  const tree = renderer.create(<MetadataElement theme={theme} />).toJSON();
  test('Then the metadata should be displayed muted and inline', () => {
    expect(tree).toMatchSnapshot();
  });
});
