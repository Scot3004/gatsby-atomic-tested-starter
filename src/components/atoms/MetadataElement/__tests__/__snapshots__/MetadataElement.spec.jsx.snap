// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When I see the metadata of a post Then the metadata should be displayed muted and inline 1`] = `
.c0 {
  color: #555;
  display: inline-block;
  margin: 0;
}

<div
  className="c0"
/>
`;
